test/performance/README.md

```diff
@@ -1,10 +1,81 @@
 # Performance tests
 
-Knative performance tests are test geared towards producing useful performance
+Knative performance tests are tests geared towards producing useful performance
 metrics of the knative system. As such they can choose to take a blackbox
 point-of-view of the system and use it just like an end-user might see it. They
 can also go more whiteboxy to narrow down the components under test.
 
-Existing scenarios:
+## Adding performance tests
```

Comment:
It would make sense to describe perf test infra first and then describe how to write new perf tests on top of it. 
1. Infra
2. how to write new test
3. how to view the result